.github/workflows/build_conda_linux.yml

```diff
@@ -164,31 +164,36 @@ jobs:
           BINARY_NAME=$(ls "$CONDA_OUTPUT_PATH"/*.tar.bz2)
           echo "$BINARY_NAME"
 
+          export CONDA_ENV_SMOKE="${RUNNER_TEMP}/pytorch_pkg_helpers_${GITHUB_RUN_ID}_smoke"
+          conda create --yes --prefix "${CONDA_ENV_SMOKE}" python="${PYTHON_VERSION}"
+          export CONDA_RUN_SMOKE="conda run -p ${CONDA_ENV_SMOKE}"
+
           if [[ "${{ inputs.package-name }}" = "torchvision" ]]; then
-              ${CONDA_RUN} conda install -y 'numpy>=1.11'
-              ${CONDA_RUN} conda install pillow
+              ${CONDA_RUN_SMOKE} conda install -y 'numpy>=1.11'
+              ${CONDA_RUN_SMOKE} conda install pillow
           fi
           if [[ "${{ inputs.package-name }}" = "torchdata" ]]; then
-              ${CONDA_RUN} conda install -y portalocker>=2.0.0
+              ${CONDA_RUN_SMOKE} conda install -y portalocker>=2.0.0
           fi
```

Comment:
Sorry, but why this is needed?

Reply:
Not sure, I think this was required for when Nova was developed at some point. Maybe we can try to remove these and see what's up